src/Commands/AdminCommands/CleanupCommand.php

```diff
@@ -327,7 +351,7 @@ private function getQueries($settings)
     public function execute()
     {
         $message = $this->getMessage();
-        $chat_id = $message->getFrom()->getId();
+        $chat_id = $message->getChat()->getId();
```

Comment:
Because... `getFrom()->getId()` would return id of the user who called this command, so the bot would reply to the private chat with that user.